### PR TITLE
WIP:Update the heating and current drive docs to include all models

### PR DESCRIPTION
**Pull request by chris-ashe**
## Checklist

I confirm that I have completed the following checks:

- [ ] I have justified any large differences in the regression tests caused by this pull request in the comments.
- [ ] I have added new tests where appropriate for the changes I have made.
- [ ] If I have had to change any existing unit or integration tests, I have justified this change in the pull request comments.
- [ ] If I have made documentation changes, I have checked they render correctly.
- [ ] I have added documentation for my change, if appropriate.

### CODE REVIEW

**Review by j-a-foster — CHANGES_REQUESTED** [review]
Other than comments, happy with NBI docs.

File: documentation/proc-pages/eng-models/heating_and_current_drive/RF/cutoff_ecrh.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+| `zeff`, $Z_{\text{eff}}$ | Effective charge |
+| `harnum` | Harmonic number |
+| `mode` | RF mode |
+
+----
+
+
+
+$$
+\mathtt{fc} = \frac{1}{2\pi}eB_{\text{T}}m_{\text{e}}
```
Comment (j-a-foster) [inline documentation/proc-pages/eng-models/heating_and_current_drive/RF/cutoff_ecrh.md:20]: I think it should be `/m_e` at the end of this equation?

**Review by j-a-foster — CHANGES_REQUESTED** [review]
Apart from comments, happy with rest of the HCD docs.

File: documentation/proc-pages/eng-models/heating_and_current_drive/NBI/nbi_overview.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+!!! info "Info" 
+    For the full table of values for $A_{ijk}$ & $B_{ijk}^{(z)}$\  please see the accompanying paper[^1] or `current_drive.py`
+
+For a plasma having an arbitrary mix of $N$ different types of impurities with densities $n$, and charges $Z_q (q = 1, ..., N)$, the beam stopping cross-section can be represented as the weighted sum of the stopping cross- sections for $N$ reference single-impurity plasmas. In each of these reference plasmas, the electron density and the proton density (including that of deuterium and tritium ions) are the same as in a true plasma. The impurity density, however, is increased in order to satisfy quasi-neutrality. The weighting function is the electron density $n_qZ_q$ associated with the aum impurity (in the true plasma), divided by the sum of these densities. The result is:
```
[the remark this answers is not in the document]
Comment (chris-ashe) [reply documentation/proc-pages/eng-models/heating_and_current_drive/NBI/nbi_overview.md:63]: Has been corrected to amu (atomic mass unit)

File: documentation/proc-pages/eng-models/heating_and_current_drive/RF/ehst_lower_hybrid.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Ehst Lower Hybrid
+- `iefrf/iefrffix` = 4
+$$
+\text{Current drive efficiency [A/W]} = \frac{T_{\text{e}}^{0.77} (0.034 + 0.196\beta)}{R_0 n_{\text{e},20}}\frac{\frac{32}{5+Z_{\text{eff}}}+2+\frac{\frac{12\left(6+Z_{\text{eff}}\right)}{5+Z_{\text{eff}}}}{3+Z_{\text{eff}}}+\frac{3.76}{Z_{\text{eff}}}}{12.507}
```
[the remark this answers is not in the document]
Comment (chris-ashe) [reply documentation/proc-pages/eng-models/heating_and_current_drive/RF/ehst_lower_hybrid.md:4]: `feffcd` is kept out of the formulas in order to just show their true form. Using `feffcd` to scale  the output is discussed in other points of the docs

File: documentation/proc-pages/eng-models/heating_and_current_drive/RF/cutoff_ecrh.md (hunk 1 above)
Comment (chris-ashe) [reply documentation/proc-pages/eng-models/heating_and_current_drive/RF/cutoff_ecrh.md:20]: Corrected

**Review by j-a-foster — APPROVED** [review]
Happy with corrections. Ready for merge.

**Pull request merged**